### DISCUSSION TITLE
Azure AD integration error

### DISCUSSION THREAD

**Discussion by Nikhilps0** [requarks/wiki#7344] [Error / Bug Report] [answered]
I have successfully configured the AD in the wiki, and login also working. after the login it is redirect to the wiki page. But I noticed that the logout option is missing after the login. Is this issue only to me or anyone faced the issue.

**Comment by heraldodinero** [comment]
Only you, I have log out. What browser are you using?

**Reply by Nikhilps0** [reply, marked as answer]
I am using it on Microsoft edge. 

<img width="245" alt="wikierror" src="https://github.com/user-attachments/assets/942d18d1-9d9a-4ff9-a598-1f087d711746">


I found the problem, the setting I used in the nginx configuration was causing it.
